src/nvim/cursor_modify.h

```diff
@@ -0,0 +1,9 @@
+#ifndef NVIM_CURSOR_MODIFY_H
+#define NVIM_CURSOR_MODIFY_H
+
+#include "nvim/vim.h"
```

Comment:
I don't think this is needed (nvim compiles fine without it)... might be wrong though.